spec/features/results/results_spec.rb

```diff
@@ -159,4 +159,122 @@
       expect(page).to have_content("50%")
     end
   end
+
+  it "User sees aggregate results from comparison with house member who is a favorite", :vcr do
+
+    HouseFavorite.create!(house_member_id: @aoc.id, user_id: @user1.id)
+
+    visit "/comparison?topic=Armed%20Forces%20and%20National%20Security&id=#{@aoc.congress_id}"
+    choose "#{@bill_1.bill_id}_yes"
+    choose "#{@bill_2.bill_id}_yes"
```

Comment:
Is this a way to click on a radio button?

Reply:
Correct! That's how you select a specific radio button with Capybara.